packages/dashboard/src/CdcDashboard.tsx

```diff
@@ -37,11 +37,100 @@ import './scss/main.scss'
 import '@cdc/core/styles/v2/main.scss'
 import AdvancedEditor from '@cdc/core/components/AdvancedEditor'
 
+export type APIFilter = {
+  apiEndpoint: string
+  valueSelector: string
+  textSelector: string
+}
+
+export type SharedFilter = {
+  type?: 'urlfilter' | 'datafilter' | ''
+  queryParameter?: string
+  active?: string
+  usedBy?: string[]
+  parent?: string
+  setBy?: string
+  columnName?: string
+  resetLabel?: string
+  showDropdown?: boolean
+  labels?: {
+    [p: string]: any
+  }
+  key: string
+  values?: string[]
+  apiFilter?: APIFilter
+  datasetKey?: string
+  tier?: number
+}
+
+type DataSet = {
+  data: any
+  dataDescription: string
+  dataUrl: string
+  runtimeDataUrl: string
+  dataFileSourceType: string
+  formattedData: any
+}
+
+type DataSets = {
+  [p: string]: DataSet
+}
+
+export type Config = DataSet & {
+  dashboard: {
+    sharedFilters: SharedFilter[]
+    datasets: DataSets
+    description: any
+    title: any
+    theme: any
+    filters: any
+  }
+  visualizations: {
+    [vizKey: string]: {
+      visualizationType: string
+      dataKey: string
+      formattedData: any
+      dataDescription: string
+      data: any
+      originalFormattedData: any
+      editing: boolean
+      type: 'chart' | 'map' | 'data-bite' | 'waffle-chart' | 'markup-include' | 'filtered-text' | 'filter-dropdowns'
+      newViz: boolean
+      hide: any[]
+      table: {
+        showDataTableLink: boolean
+      }
+      general: any
+      title: string
+      usesSharedFilter: any
+    }
+  }
+  datasets: DataSets
+  dataFileName: string
+  table: any
+  rows: any[]
+  data: any
+  filterBehavior: string
+}
+
+type APIFilterDropdowns = {
+  // null means still loading
```

Comment:
this type is not exported and is not relevant to any other component than CdcDashboard. I think this one should remain in this file.